js/src/views/map/map.js

```diff
@@ -18,48 +21,120 @@ var MapView = Backbone.View.extend({
       worldCopyJump: true
     },
     cartodb: {
-      user_name: 'dhakelila',
-      type: 'cartodb',
+      user_name: 'goal16',
```

Comment:
Wouldn't be better use the CONFIG json file for these cases? I think that's the purpose of this.